<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Materiel</class>
 <widget class="QDialog" name="Materiel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size :20px;

}
 QTabWidget{
background:#333;
border-radius:15px;
}

QLabel {
postion:center;
color :#CD5C5C;
}
QTableView {
background:white;
border-radius:15px;
font-size :20px;

}
QPushButton {
background: #CD5C5C;border:none;
color:white;
 border-radius: 15px;
}
QPushButton:hover
{ color:white;
 border-radius: 15px;
background-color :#F08080;border:none;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>1101</width>
     <height>611</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste des materiels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Liste des materiels</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher un materiel </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Avec</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>882</x>
       <y>190</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>901</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*QTableView::item
{
anchors.fill: racesWindow
        width: 840
        height: 400
       
}
*/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_line">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Ajouter un materiel</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Code materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>La date d'acquisition</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Type materiel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeLine">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="etatLine">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque materiel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="marqueLine">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>510</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat materiel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="etat">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Modifier un materiel</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Code materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>La date d'acquisition</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="etatLine_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Type materiel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="marqueLine_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque materiel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit_5">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>480</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Supprimer un materiel</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrer le code du materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/téléchargement.png</normaloff>../../Desktop/téléchargement.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
